src/pages/Loopstudio/HeroSection/HeroSection.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useState } from 'react';
+import logo from '../../../assets/loopstudios-img/logo.svg';
+import hamburger from '../../../assets/loopstudios-img/icon-hamburger.svg';
+import close from '../../../assets/loopstudios-img/icon-close.svg';
+import './HeroSection.css';
+
+function HeroSection() {
+  const [nav, showNav] = useState(false);
+  const handleNav = () => {
+    showNav(!nav);
+  };
+  return (
+    <div className="hero--section relative ">
+      {nav && (
+        <nav className="flex pt-2 justify-between px-3 items-center  ">
+          <img className="z-10" src={logo} alt="" />
+          <img className="h-4" onClick={handleNav} src={close} alt="" />
+          <img className="close--btn " onClick={handleNav} src={close} alt="" />
+          <ul className="mobile--nav absolute top-0 left-0  flex flex-col justify-center text-3xl uppercase pl-4 tracking-widest bg-black text-white ">
+            <li>About</li>
+            <li>Careers</li>
+            <li>Events</li>
+            <li>Products</li>
+            <li>Support</li>
+          </ul>
+        </nav>
+      )}
+      <nav className="flex pt-2 justify-between px-3 items-center md:pt-6 md:px-10 lg:px-20 ">
```

Comment:
Instead of hiding having two nav components with a minor difference - reuse only one: one suggestion is to have an exhaustive list of classes which only change between md and smaller resolutions.